dipy/workflows/reconst.py

```diff
@@ -1022,10 +1034,13 @@ def run(self, input_files, bvalues_files, bvectors_files, mask_files,
     def get_fitted_ivim(self, data, mask, bval, bvec, b0_threshold=50):
         logging.info('Intra-Voxel Incoherent Motion Estimation...')
         bvals, bvecs = read_bvals_bvecs(bval, bvec)
-        if b0_threshold < bvals.min():
-            warn("b0_threshold (value: {0}) is too low, increase your "
-                 "b0_threshold. It should be higher than the first b0 value "
-                 "({1}).".format(b0_threshold, bvals.min()))
+        # If all b-values are smaller or equal to the b0 threshold, it is
+        # assumed that no thresholding is requested
+        if any(mask_non_weighted_bvals(bvals, b0_threshold)):
+            if b0_threshold < bvals.min():
+                warn("b0_threshold (value: {0}) is too low, increase your "
+                     "b0_threshold. It should be higher than the first b0 "
+                     "value ({1}).".format(b0_threshold, bvals.min()))
```

Comment:
Can you change this using f-string? thanks!

Reply:
That is a separate commit and a separate PR: it does not help addressing the warnings.